docs_app/content/tutorial/1-subscribing.md

```diff
@@ -0,0 +1,66 @@
+# 1. Subscribing to an Observable
+
+## TLDR:
+
+- Observables don't do anything until you call [subscribe](API)
+- Calling [subscribe](API) will synchronously execute code that sets up the subscription, telling the underlying [producer](GL) to start pushing values to your code (the [consumer](GL)).
+- [subscribe](API) may be called with _one function_ that is the [next](GL) handler, *OR* an [`Observer`](API) object, which has a [next](GL), [error](GL), and/or [complete](GL) handler on it.
+- If no [error](GL) handler is provided, RxJS will assume all errors that are pushed are "unhandled" errors, and rethrow them on a different call stack.
+
+## Overview
+
+Chances are, the first time you encounter an [Observable](API), it is because you have had one returned to you by some other API. In this case, the most helpful thing for you to know is how to get values from the observable, and the semantics of [subscription](GL)
+
+> The first thing to know is, an observable doesn't "do" anything.
+
+An observable, in fact, is _not_ a "stream". An observable is a template for [subscription](GL), through which your code (the [consumer](GL)), will tell a [producer](GL) to start pushing values to it by subscribing to an observable, creating a [subscription](GL).
+
+## Basic Subscription
+
+```ts
+const source$ = getSomeObservable();
+
+source$.subscribe(value => {
+    // do something with each value as it is pushed to your code
+    console.log(value);
+});
+```
+
+The most basic form of subscription is to subscribe to the observable with a single function as the argument to [`subscribe`](API). This will execute the observable's initialization function and create the [subscription](GL).
+
+## Subscription With An Observer
+
+```ts
+const source$ = getSomeObservable();
+
+source$.subscribe({
+    next(value) {
+        // Do something with each value as it is pushed to your code
+        console.log(value);
+    },
+    error(err) {
+        // Whatever is producing and pushing values has encountered an error
+        // and will no longer send values
+        console.error(err);
+    },
+    complete() {
+        // The producer pushing values is done pushing values, and wishes to notify
+        // your code that you will no longer receive values.
+        console.log('done');
+    }
+})
+```
+
+An `Observer` is a type, basically any object with a `next`, `error`, and `complete` method on it. A "partial observer" is any object with at least one of those methods. It is a way to pass these three handlers to the subscription so that a [producer](GL) can notify your code (the [consumer](GL)). `next` is called when the producer pushes a value to the consumer. `error` is called when the producer has encountered an error and must stop sending values, and `complete` is called  when the producer has pushed all values to the consumer, and will push no more.
+
```

Comment:
One of the very common SO questions with regard to RxJS basics is the order of execution. Would it be worth adding a section here to cover that? Something like this:

## Order of Execution
Because the next, error, and complete methods are callbacks, they are not executed until they are called back either when the next item is emitted, an error occurs, or the producer is complete. That means that the any code within these methods is not immediately executed.

In the following example, the source$.subscribe statement is executed and then the `console.log(this.localValue)` statement. At that point, the `this.localValue` variable is not yet set. When the producer pushes the first value, the next method is executed, and `this.localValue` is set to the emitted value.

Example (not syntax checked):
```
const source$ = getSomeObservable();
localValue: string;

source$.subscribe({
    next(value) {
        // Do something with each value as it is pushed to your code
       this.localValue = value;
        console.log(value);
    },
    error(err) {
        // Whatever is producing and pushing values has encountered an error
        // and will no longer send values
        console.error(err);
    },
    complete() {
        // The producer pushing values is done pushing values, and wishes to notify
        // your code that you will no longer receive values.
        console.log('done');
    }
})
// this.localValue is undefined here
console.log(this.localValue);
```

Reply:
I'm not sure I would say this exactly... 

```ts
function getSomeObservable() {
  return of(1, 2, 3, 4, 5);
}
```

In that case, `console.log(this.localValue)` would log `5`, as the returned observable will synchronously push values when it is subscribed to.

Reply:
The above may be a great topic to include somewhere? Maybe even in this "order of execution" (or otherwise named) section that stresses that these are callbacks *and* mentions how Observables behave synchronously vs asynchronously?